.circleci/local_publish_helpers.sh

```diff
@@ -72,7 +72,7 @@ function generatePkgCli {
   cp package.json ../build/node_modules/package.json
   if [[ "$CIRCLE_BRANCH" == "release" ]] || [[ "$CIRCLE_BRANCH" =~ ^run-e2e-with-rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^release_rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
     # This will generate a file our arm64 binary
-    npx pkg --no-bytecode -t node14-linux-arm64 ../build/node_modules -o ../out/amplify-pkg-linux-arm64
+    npx pkg --no-bytecode --public-packages "*" --public -t node14-linux-arm64 ../build/node_modules -o ../out/amplify-pkg-linux-arm64
```

Comment:
The https://github.com/vercel/pkg#other-considerations mentions `--public-packages`
What does `--public` do here?

Reply:
This is inspired from here : https://github.com/vercel/pkg#targets

I can check if this needed.

My theory is only  ```--public-packages "*"``` is needed  because some commit might have  a dependency  on a package with a broken license in its package.json